OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -74,11 +74,6 @@ public override Activity Tick(Actor self)
 				self.QueueActivity(mobile.MoveTo(moveTo, 1));
 				self.SetTargetLine(Target.FromCell(self.World, moveTo), Color.Gray, false);
 
-				// TODO: The harvest-deliver-return sequence is a horrible mess of duplicated code and edge-cases
```

Comment:
These notifications are correct in the context of what this code branch is trying to do: if no resources are found then the harvester should move to its assigned/assumed resource field instead of clogging up the base.  Having the unit drive there when it would normally use a carryall is unexpected.

This entire "get out of the way" feature, on the other hand, is of questionable use (there are other mechanisms designed to help unclog refineries) and is contributing to the overall mess of the harvester code.  If we're going to remove these notifications then we should go further and remove the whole code branch.  If this regresses gameplay in a meaningful way then we can reimplement it in a more maintainable way.